### PR TITLE
🌱 docs(footer): Add trademark disclaimer

### LINKED ISSUE #2735: Website does not have the correct trademark disclaimer
As part of our ongoing effort to https://github.com/cncf/techdocs/issues/198, we noticed that the website does not pass the trademark criteria on [CLOMonitor](https://clomonitor.io/search?foundation=cncf&not_passing_check=trademark_disclaimer&page=1).

To fix this:
Head to the source code of the website. In the `<footer>` section, add a **disclaimer** or **link** to the Linux foundation trademark disclaimer page:

**Disclaimer**
```
<footer>
   <p>The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, 
         please see our <a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademark Usage page</a>.
   </p>
</footer>
```

**Link**
```
 <footer>
      <ul>
          <li><a href="https://www.linuxfoundation.org/legal/trademark-usage">Trademarks</a></li>
      </ul>
 </footer>
```

### PR DESCRIPTION
**Pull request by mariasalcedo**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Adds correct trademark disclaimer, as part of our ongoing effort to https://github.com/cncf/techdocs/issues/198

**Which issue(s) this PR fixes**:
Fixes #2735 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by mikemorris** [comment]
Looks reasonable to me, thanks for taking care of this @mariasalcedo!

/ok-to-test
/lgtm

**Comment by mariasalcedo** [comment]
@mikemorris Thanks! Please add the `approved`  label to be able to merge it.

File: site-src/overrides/partials/copyright.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright (c) 2016-2024 Martin Donath <martin.donath@squidfunk.com>
```
Comment (robscott) [inline site-src/overrides/partials/copyright.html:2]:
```suggestion
  Copyright (c) Kubernetes Authors and Martin Donath <martin.donath@squidfunk.com>
```

Wasn't sure how to handle this, but got some [help in Slack](https://kubernetes.slack.com/archives/C1TU9EB9S/p1717616676425929?thread_ts=1717613057.828109&cid=C1TU9EB9S) and this was the suggestion from @jberkus.

File: site-src/overrides/partials/copyright.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  {% if not config.extra.generator == false %}
+    Made with
+    <a
+      href="https://squidfunk.github.io/mkdocs-material/"
+      target="_blank" rel="noopener"
+    >
+      Material for MkDocs
+    </a>
+  {% endif %}
+</div>
```
Comment (robscott) [inline site-src/overrides/partials/copyright.html:40]: Please leave a trailing newline here.

**Review by robscott — COMMENTED** [review]
Thanks @mariasalcedo! Unfortunately we're going to need to hold off on this until https://github.com/cncf/foundation/issues/809 is resolved.

/hold

**Comment by youngnick** [comment]
LGTM pending resolution of the license question.

**Comment by shaneutt** [comment]
This is blocked on https://github.com/cncf/foundation/issues/809 and should be frozen to avoid auto-close.

/lifecycle frozen

**Comment by robscott** [comment]
Now that https://github.com/cncf/foundation/issues/809 has been approved, this should be good to go.

/hold cancel
/lgtm
/approve
/retest

**Pull request merged**